src/gedcom.ts

```diff
@@ -148,9 +168,16 @@ function createIndi(
   const indi: JsonIndi = { id, fams };
 
   // Name.
-  const nameTag = findTag(entry.tree, 'NAME');
-  if (nameTag) {
-    const { firstName, lastName } = extractName(nameTag.data);
+  const nameTags = findTags(entry.tree, 'NAME');
+  const isMaiden = (nameTag: GedcomEntry) => {
+    const type = findTag(nameTag.tree, 'TYPE');
+    return type !== undefined && type.data === 'maiden';
+  };
+  const main = nameTags.find(x => !isMaiden(x));
```

Comment:
If the only NAME tag provided is marked as *maiden*, we should also use it as the main name.
```typescript
const main = nameTags.length && (nameTags.find(x => !isMaiden(x)) || nameTags[0]);
```

Reply:
I don't think it is correct to use maiden last name as the main. at least this is something that does not make sense to me.
1. user does not know the main lname, but knows maiden - we should not treat it as main as it adds an inconsistency
2. user added the main lname as maiden name... lib should not take responsibility for that imho
3. ???